cmake/targets.cmake

```diff
@@ -459,10 +459,12 @@ function(wn_filesystem_files name)
       string(SUBSTRING ${full_source} 0 ${start_of_logical} root_dir)
       list(APPEND PYTHON_ARGS "--directory" ${root_dir} "--files" ${logical_source})
     endforeach()
+    find_package(Python3 COMPONENTS Interpreter REQUIRED)
+    message(FATALE_ERROR ${Python3_EXECUTABLE})
```

Comment:
I'm guessing this is supposed to be `message(FATAL_ERROR ${Python3_EXECUTABLE})`

Reply:
Actually should this e here at all?

Reply:
You can probably remove the `message(FATALE_ERROR ${Python3_EXECUTABLE})` now @AWoloszyn.